CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ## Unreleased
 - Add `AttributeValueDouble` to `AttributeValue`.
-
+- Add an API `Functions.returnToString()`.
```

Comment:
Don't think this really deserves a changelog change. But up to you.

Reply:
I'll prefer to leave it in the CHANGELOG since it's a public API. Later when people want to write new trace exporters, this API may be useful.